build/build-images.sh

```diff
@@ -24,7 +24,8 @@ source $ROOT/dev/util.sh
 
 # if parallel utility is installed, the docker build commands will be parallelized
 if command -v parallel &> /dev/null ; then
```

Comment:
I think it might be better to run the `else` even if `NUM_BUILD_PROCS` is defined but it's set to 1; something like this should work:

```bash
if command -v parallel &> /dev/null && [ -n "$NUM_BUILD_PROCS" ] && [ "$NUM_BUILD_PROCS" != "1" ]; then
  ...
else
  ...
fi
```

If we do it this way, we could also use `$NUM_BUILD_PROCS` directly and remove the `num_procs=` line.

what do you think?

Reply:
Yes, I agree. I'm making this change.